lib/models/device.coffee

```diff
@@ -249,6 +249,41 @@ exports.getApplicationName = (uuid, callback) ->
 	exports.get(uuid).get('application_name').nodeify(callback)
 
 ###*
+# @summary Get application container id
```

Comment:
Good catch Juan, I originally had a `getContainerId()` endpoint and missed this part when refactoring.